packages/e2e-tests/specs/editor/various/embedding.test.js

```diff
@@ -171,7 +171,7 @@ describe( 'Embedding content', () => {
 	it( 'should render embeds in the correct state', async () => {
 		// Valid embed. Should render valid figure element.
 		await insertEmbed( 'https://twitter.com/notnownikki' );
-		await page.waitForSelector( 'figure.wp-block-embed-twitter' );
+		await page.waitForSelector( 'figure.wp-block-embed' );
```

Comment:
Now that we are sure to preserve `wp-block-embed-xyz` classes, maybe we can revert the changes in this test file and remove the snapshots? In general, it gives me confidence that we're able to preserve these tests throughout a refactor :)

Reply:
I believe that having the snapshot isn't a bad thing here. In addition:
>Previously this class was added before save as default block name class. With your previous comment for backwards compatibility I added it in save function. So it will not be visible before save but the final classes are identical (after save).

If you think this should really change and remove the snapshots I will have to change the tests to save the post and then check for the class. Just tell me to change it this will be the direction.

Reply:
OK, not worth the effort then :)